lib/controller/admin/appearance.php

```diff
@@ -254,33 +254,33 @@ protected function updateAutoTeaserConfiguration() {
 
 		if (!is_numeric($percentage) or $percentage % 1 > 0 or 1 > $percentage or $percentage > 100) {
 			$success = FALSE;
-			$errorMessages[$percentageFieldName] = __('Percentage must be a positive integer between 1 and 100', 'laterpay');
+			$errorMessages[$percentageFieldName] = tx_laterpay_helper_string::tr('Percentage must be not negative integer and have value between 1 and 100', 'laterpay');
 			$errors[] = $percentageFieldName;
 		}
 
 		if (!is_numeric($minWords) or $minWords < 0 ) {
 			$success = FALSE;
-			$errorMessages[$minWordsFieldName] = __('Min count of words must be a positive integer greater than 0', 'laterpay');
+			$errorMessages[$minWordsFieldName] = tx_laterpay_helper_string::tr('Min count of words must be not negative integer more than 0', 'laterpay');
 			$errors[] = $minWordsFieldName;
 		}
 
 		if (!is_numeric($maxWords) or $maxWords < 0 ) {
 			$success = FALSE;
-			$errorMessages[$maxWordsFieldName] = __('Max count of words must be a positive integer greater than 0', 'laterpay');
+			$errorMessages[$maxWordsFieldName] = tx_laterpay_helper_string::tr('Max count of words must be not negative integer more than 0', 'laterpay');
```

Comment:
I fixed the English as you can see in the line above. This destroys that fix again.